src/plugins/Camera.js

```diff
@@ -60,73 +58,9 @@ module.exports = class Camera extends OverlayPlugin {
             this.toolbarBtn.className += ' gm-disabled-widget-pop-up';
             this.toolbarBtnImage.className += ' gm-disabled-widget-icon';
         } else {
-            this.toolbarBtn.onclick = this.toggleStreaming.bind(this);
-        }
-    }
-
-    /**
-     * Toggle local video forward forwarding.
-     * Redirect the client webcam video stream to the instance.
-     */
-    toggleStreaming() {
-        if (!this.streaming) {
-            this.startStreaming();
-        } else {
-            this.stopStreaming();
+            this.toolbarBtn.onclick = () => {
```

Comment:
maybe i wrong but i think 

`this.toolbarBtn.onclick = this.instance.mediaManager.toggleVideoStreaming`

Could get the job. Here we bind an arrow fn which call a fn.